storage_service/common/signals.py

```diff
@@ -111,13 +111,25 @@ def ldap_populate_user_profile(sender, user=None, ldap_user=None, **kwargs):
     if user.is_superuser:
         return
 
-    role = roles.USER_ROLE_READER
-    if settings.AUTH_LDAP_ADMIN_GROUP in ldap_user.group_names:
-        role = roles.USER_ROLE_ADMIN
-    elif settings.AUTH_LDAP_MANAGER_GROUP in ldap_user.group_names:
-        role = roles.USER_ROLE_MANAGER
-    elif settings.AUTH_LDAP_REVIEWER_GROUP in ldap_user.group_names:
-        role = roles.USER_ROLE_REVIEWER
+    role = roles.settings.DEFAULT_USER_ROLE
+
+    if hasattr(settings, "AUTH_LDAP_GROUP_SEARCH"):
```

Comment:
Adding a conditional to skip the following when not configuring the GROUP_SEARCH fixes the issue and makes optional the group settings.